audio-worklet/bitcrusher.html

```diff
@@ -42,33 +26,40 @@
   <head>
     <title>Bitcrusher</title>
     <link rel="stylesheet" href="style.css">
+    <script src="src/bitcrusher.js"></script>
+    <script src="src/sourcecontroller.js"></script>
```

Comment:
nit: If your class name is `SourceController`, the filename usually becomes `source-controller.js`.